README.md

```diff
@@ -109,12 +109,23 @@ $ cmake -DENABLE_COMPLEX8=ON ..
 ```
 Reduce to float accuracy for complex numbers. Requires half as much RAM (1 additional qubit). Compatible with SSE 1.0 and single precision accelerator devices.
 
+## On-Chip Hardware Random Number Generation 
+
+```
+$ cmake -DENABLE_RDRAND=OFF ..
+```
+Turn off the option to attempt using on-chip hardware random number generation, which is on by default. If the option is on, Qrack might still compile to attempt using hardware random number generation, but fall back to software generation if the RDRAND opcode is not actually available. Some systems' compilers, such as that of the Raspberry Pi 3, do not recognize the compilation flag for enabling RDRAND, in which case this option needs to be turned off.
+
+Relatively recent Intel processors may provide the RDRAND (and RDSEED) processor operation codes, for hardware random number generation. The authors of Qrack cannot promise anything about the security or quality of these random numbers. (It is not hard to modify Qrack to use an entropy source you personally trust.) However, there is reason to suspect based on descriptions of the on-chip generator that the conditioned hardware entropy stream typically has a high von Neumann entropy content, (depending partly on load,) that this is "quantum" or "true" random entropy.
+
+There is little or no obvious benefit or detriment, in choosing between deterministic and nondeterminstic sources of high quality random numbers, for quantum computation simulation. However, it is the opinion of one of the authors of Qrack, that seemingly the one aspect of a quantum computer that cannot be emulated even in principle by a deterministic machine is "unitarity breaking," "true random numbers," whether this makes any measurable or falsifiable difference.
```

Comment:
This paragraph is super awkward to read.

Reply:
Will revise.

Reply:
(I just cut it.)